Sources/Common/DataModel/AbstractPointLocator/index.js

```diff
@@ -0,0 +1,55 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkLocator from 'vtk.js/Sources/Common/DataModel/Locator';
+
+function vtkAbstractPointLocator(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkAbstractPointLocator');
+
+  publicAPI.setBounds = (input) => {
+    model.bounds[0] = input[0];
+    model.bounds[1] = input[1];
+    model.bounds[2] = input[2];
+    model.bounds[3] = input[3];
+    model.bounds[4] = input[4];
+    model.bounds[5] = input[5];
+  };
+
+  publicAPI.getBounds = (output) => {
+    output[0] = model.bounds[0];
+    output[1] = model.bounds[1];
+    output[2] = model.bounds[2];
+    output[3] = model.bounds[3];
+    output[4] = model.bounds[4];
+    output[5] = model.bounds[5];
+  };
```

Comment:
you should be able to get those 2 accessors automatically with `macro.getSetArray(...)`